### ISSUE TITLE
Mock-ToolRunner fails to emit stdline.

### ISSUE THREAD

**Issue by ILMTitan** [Microsoft/vsts-task-lib#227]
ToolRunner will emit to the event 'stdline'. Mock-ToolRunner does not. Tasks that rely on the 'stdline' event are untestable.

**Comment by bryanmacfarlane** [comment]
This is a task-lib issue right?  Is there a need to create issue here as well?

**Comment by ILMTitan** [comment]
Correct, it is an issue with vsts-task-lib. If adding the issue to more
places will get it taken care of, I would be in favor of that.

**Comment by damccorm** [comment]
Added PR to address this.
